network/src/protocols/discovery/mod.rs

```diff
@@ -357,18 +356,26 @@ where
                     assert_ne!(peer_id, self_peer_id);
                     // Update internal state of the peer with new Note.
                     self.known_peers.insert(peer_id, (peer_info.clone(), note));
+
+                    // The multiaddrs in the peer's discovery Note.
+                    let mut peer_addrs: Vec<Multiaddr> = peer_info
```

Comment:
@phlip9 Should we create a function to get `Vec<Multiaddr>` from `PeerInfo`?

Reply:
I think this should be a member of the "verified" struct